### DISCUSSION TITLE
Locais de trabalho e incentivos a Open Source

### DISCUSSION THREAD

**Discussion by yanmagale** [frontendbr/forum#1241] [Discussão]
Olá pessoal, tudo bem?

Gostaria de saber se na empresa que vocês trabalham (ou em empresas que já trabalharam) se existe algum tipo de cultura incentivada para Open Source. Se vocês possuíam algum tempo para trabalhar em projetos abertos, hackathons internos e coisas do tipo.

Como é(foi) a experiência? Como era a cultura?

**Comment by valterbarros** [comment]
Aqui onde trabalho existe alguns projetos open source principalmente na parte de desenvolvimento mobile android e ios, esses projetos são libs e utilizamos internamente em nossos apps. Mas tipo uma coisa que eu acho legal que tenho vontade de fazer e que não é feito aqui é contribuir para projetos open source usados no dia a dia alguma lib ou framework.

**Comment by klarkc** [comment]
Aqui na empresa contribuímos com Open Source sempre que possível, além disso o pessoal é incentivado a participar de meetup's seja contribuindo com conteúdo ou aprendendo com a comunidade (inclusive sediamos os meetups do WordPress em Florianópolis). Em nosso GitHub publicamos a maior parte das ferramentas que usamos internamente, acredito que é uma ótima forma de mostrar o nosso trabalho e de devolver um pouco do que ganhamos da comunidade.

Se a empresa onde trabalha não tem essa cultura, mas tem uma cultura de colaboração, colocar essa ideia para todos os devs, mencionando todas essas vantagens não é difícil - certamente será bem aceito.
